### ISSUE TITLE
could not use python script

### ISSUE THREAD

**Issue by includewins0ck2** [pawelsalawa/sqlitestudio#4906]
Can't use the python plugin, I've tested it many times on a pure win11 system, no matter what version of python is installed, whether it's x86, or x64, with sqlitestudio 3.44 it all tells me It can't load the ScriptingPython.dll, but the file is clearly there.

**Comment by pawelsalawa** [comment]
Do you have Python installed in your system? Is it added to PATH?

**Comment by includewins0ck2** [comment]
> Do you have Python installed in your system? Is it added to PATH?

Yeah, I'm pretty sure. I installed a clean environment with vmware and tested it many times.

**Comment by tuffnatty** [comment]
Is it Python 3.9?

**Comment by includewins0ck2** [comment]
> Is it Python 3.9?
Thank you so much, I tried 3.8, 3.10, 3.11 but not 3.9, now it's solved, thanks!
